### ISSUE TITLE
Mod version 1.4.7 bug: Shotgun refuses to fire for some reason

### ISSUE THREAD

**Issue by Yokisan249** [Hypick122/BetterShotgun#41]
### Describe the bug

Previously the shotgun was working perfectly fine until I installed Lethal Level Loader mod for an SCP dungeon mod. Now it only says this in the log:

[Error  : Unity Log] MissingMethodException while executing 'performed' callbacks of 'Movement/ActivateItem[/Mouse/leftButton]'
[Info   : Unity Log] Calling shoot gun....
[Error  : Unity Log] MissingMethodException: Method not found: int .RoundManager.PlayRandomClip(UnityEngine.AudioSource,UnityEngine.AudioClip[],bool,single,int)
Stack trace:
BetterShotgun.ShotgunPatch.ReplaceShotgunCode (ShotgunItem __instance, UnityEngine.Vector3 shotgunPosition, UnityEngine.Vector3 shotgunForward) (at <d8520ab0c4ce4f5ebdc7a0d3ec3cc9a8>:IL_0001)
(wrapper dynamic-method) ShotgunItem.DMD<ShotgunItem::ShootGun>(ShotgunItem,UnityEngine.Vector3,UnityEngine.Vector3)
ShotgunItem.ShootGunAndSync (System.Boolean heldByPlayer) (at <255c0877f9414c3aa6f1fe5079ac13f3>:IL_0083)
(wrapper dynamic-method) ShotgunItem.DMD<ShotgunItem::ItemActivate>(ShotgunItem,bool,bool)
GrabbableObject.UseItemOnClient (System.Boolean buttonDown) (at <255c0877f9414c3aa6f1fe5079ac13f3>:IL_005C)
GameNetcodeStuff.PlayerControllerB.ActivateItem_performed (UnityEngine.InputSystem.InputAction+CallbackContext context) (at <255c0877f9414c3aa6f1fe5079ac13f3>:IL_008B)
UnityEngine.InputSystem.Utilities.DelegateHelpers.InvokeCallbacksSafe[TValue] (UnityEngine.InputSystem.Utilities.CallbackArray`1[System.Action`1[TValue]]& callbacks, TValue argument, System.String callbackName, System.Object context) (at <dbb0ce7b96144bd8bcbcb8b0d765f970>:IL_001B)
UnityEngine.InputSystem.LowLevel.<>c__DisplayClass7_0:<set_onUpdate>b__0(NativeInputUpdateType, NativeInputEventBuffer*)
UnityEngineInternal.Input.NativeInputSystem:NotifyUpdate(NativeInputUpdateType, IntPtr)

You can see the full log here:
[LogOutput.log](https://github.com/Hypick122/BetterShotgun/files/15327648/LogOutput.log)

### Reproducible Test

Apparently this happens when you install "SCPFoundationDungeon" Mod that includes the "Lethal Level Loader" mod. Before installing those mods the shotgun worked normally but now it refuses to shoot.

### Anything else?

Edit: After removing the scp mod and its dependancies, the mod still doesnt work which only means other mods are stopping the shotgun to work properly. After testing with every mod, I realized the issue is actually caused by "SCP_Containment_Breach_AudioPack" mod that apparently did not make or specify a custom sound for the shotgun when shooting so the game just throws an error into the console when attempting to fire.

So in summary: Just ignore this post and delete it, I should have suspected earlier that it was a mod incompatibility with another.
